RecoTracker/PixelLowPtUtilities/src/StripSubClusterShapeTrajectoryFilter.cc

```diff
@@ -184,13 +184,13 @@ StripSubClusterShapeFilterBase::StripSubClusterShapeFilterBase(const edm::Parame
 
 StripSubClusterShapeFilterBase::~StripSubClusterShapeFilterBase() {
 #if 0
-    std::cout << "StripSubClusterShapeFilterBase " << label_ <<": called        " << called_ << std::endl;
-    std::cout << "StripSubClusterShapeFilterBase " << label_ <<": saturated     " << saturated_ << std::endl;
-    std::cout << "StripSubClusterShapeFilterBase " << label_ <<": test          " << test_ << std::endl;
-    std::cout << "StripSubClusterShapeFilterBase " << label_ <<": failTooNarrow " << failTooNarrow_ << std::endl;
-    std::cout << "StripSubClusterShapeFilterBase " << label_ <<": passTrim      " << passTrim_ << std::endl;
-    std::cout << "StripSubClusterShapeFilterBase " << label_ <<": passSC        " << passSC_ << std::endl;
-    std::cout << "StripSubClusterShapeFilterBase " << label_ <<": failTooLarge  " << failTooLarge_ << std::endl;
+  std::cout << "StripSubClusterShapeFilterBase " << label_ << ": called        " << called_ << std::endl;
+  std::cout << "StripSubClusterShapeFilterBase " << label_ << ": saturated     " << saturated_ << std::endl;
+  std::cout << "StripSubClusterShapeFilterBase " << label_ << ": test          " << test_ << std::endl;
+  std::cout << "StripSubClusterShapeFilterBase " << label_ << ": failTooNarrow " << failTooNarrow_ << std::endl;
+  std::cout << "StripSubClusterShapeFilterBase " << label_ << ": passTrim      " << passTrim_ << std::endl;
+  std::cout << "StripSubClusterShapeFilterBase " << label_ << ": passSC        " << passSC_ << std::endl;
+  std::cout << "StripSubClusterShapeFilterBase " << label_ << ": failTooLarge  " << failTooLarge_ << std::endl;
```

Comment:
why is this change of formatting necessary?

Reply:
This was automatically suggested by 
```
scram build code-checks
scram build code-format
```

Reply:
OK. Apparently no-one touched this file in ages ...